external-crates/move/crates/move-package/src/compilation/compiled_package.rs

```diff
@@ -354,6 +357,35 @@ impl OnDiskCompiledPackage {
             std::fs::read_to_string(&compiled_unit.source_path)?.as_bytes(),
         )
     }
+
+    fn save_disassembly_to_disk(
+        &self,
+        package_name: Symbol,
+        unit: &CompiledUnitWithSource,
+    ) -> Result<()> {
+        let root_package = self.package.compiled_package_info.package_name;
+        let bytecode_modules_dir = CompiledPackageLayout::CompiledModules.path();
```

Comment:
I'm of two minds on this. I think I'm fine with this, but part of me thinks this might be nice to be in its own separate top-level dir under the `build/<pkg_name>` (i.e., a sibling of `bytecode_modules`/`sources` (Etc)).

Thoughts? I'm pretty open to either option I think

Reply:
Actually, it's a great point. Putting them in a separate directory for trace-debugger to pick up will make it easier down the line - we can have other methods of generating disassembly (e.g., from the replay tool) to keep the same directory structure